app/src/main/res/layout/fragment_home.xml

```diff
@@ -39,16 +39,23 @@
             app:srcCompat="@drawable/wolox_logo"
             android:contentDescription="TODO" />
 
-        <androidx.appcompat.widget.Toolbar
-            android:id="@+id/toolbar"
+        <com.google.android.material.tabs.TabLayout
+            android:id="@+id/tabLayout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="0dp"
-            android:background="?attr/colorPrimary"
-            android:minHeight="?attr/actionBarSize"
-            android:theme="?attr/actionBarTheme"
+            android:layout_marginTop="16dp"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="@+id/headerSeparator" />
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/imageView">
+
```

Comment:
you can delete this empty line and change the way the component finish: 
instead of 
```
 app:layout_constraintTop_toBottomOf="@id/imageView">
</com.google.android.material.tabs.TabLayout>
```
just
```
app:layout_constraintTop_toBottomOf="@id/imageView" />
```